base/logging.jl

```diff
@@ -274,10 +274,76 @@ end
 
 default_group(file) = Symbol(splitext(basename(file))[1])
 
+function issimple(@nospecialize val)
+    val isa String && return true
+    val isa Symbol && return true
+    val isa QuoteNode && return true
+    val isa Number && return true
+    if val isa Expr
+        val.head === :quote && val[1] isa Symbol && return true
+        val.head === :inert && return true
```

Comment:
Do you want to catch tuple and vector and matrix literals where the head is `vect` `tuple` etc,
and the args are all themselves `issimple`

Reply:
not really, no